Gemfile

```diff
@@ -10,7 +10,6 @@ group :development do
   gem 'psych', '3.3.2'  # See https://github.com/faker-ruby/faker/issues/2330
```

Comment:
Don't we need to specify the dependencies on the `.gemspec` file and include here just a call to `gemspec` to avoid duplicating the definition?

Reply:
Oh good catch, juwelier used to make the gemspec out of this file, but that won't be happening anymore.